adapter/slack.py

```diff
@@ -22,6 +22,7 @@ def __init__(
         self.bot_service = bot_service
         self.handler = SlackRequestHandler(self.app)
         self.logger = logger.bind(service="SlackAdapter")
+        self.app_user_id = self.app.client.auth_test()['user_id']

```

Comment:
Can you share what this field is supposed to be?

Reply:
It's for getting the user id of the bot. Basically I needed a way to check if the one who started the thread is the bot itself. So, with this user id and the user id in the message event, I can check if the message is a reply inside a thread of the `/ask` command.

Reply:
Oh that's cool. A pretty clever way of handling it. All good then